app/views/layouts/_crowdfunding.html.haml

```diff
@@ -2,6 +2,6 @@
 - if daysleft > 0
   .crowdfunding-banner
     Help us share free growing information with the world.
-    = link_to "Support our crowdfunding campaign.", "https://www.indiegogo.com/projects/growstuff/x/6079859"
+    = link_to 'Support our crowdfunding campaign.', 'https://www.indiegogo.com/projects/growstuff/x/6079859'
     There are only #{daysleft.to_i} days left.
 
```

Comment:
Do we even need this any more, or should we strip it out? Are we likely to do a future similar fundraising effort and reuse it?